clusterloader2/testing/load/deployment.yaml

```diff
@@ -24,6 +27,15 @@ spec:
           requests:
             cpu: 10m
             memory: "10M"
+        volumeMounts:
+          {{if and $EnableConfigMaps (IsModIn .Index 20 8 19) }} # .Index % 20 in {8,19} - 10% deployments will have ConfigMap
```

Comment:
Given that we're randomly distributing instances (indexes) over time, I think "IsDivisible" would actually be enough and easier to follow.

Reply:
My intention was to have 10% of pods with secrets, 10% of pods with configmaps and to  exercise all combinations, i.e. to have:
1. a pod with just secret
2. a pod with just configmap
3. a pod with both

While I agree that it may be hard to parse at the first glance, it allows to easily express arbitrary configurations. 

Let's discuss this tomorrow in person.

Reply:
OK - I see your point.
Then my question is why do we need "InSet"? Why "IsModEqual" not enough?

Reply:
Realized that built-in eq supports multiple values. In that cases implementing just Mod is enough, which I did.